src/main/java/seedu/address/MainApp.java

```diff
@@ -70,8 +70,6 @@ public void init() throws Exception {
         logic = new LogicManager(model);
 
         ui = new UiManager(logic);
-
-        model.initialize();
     }
```

Comment:
Is this function replaced somewhere else?

Reply:
yea line 68

Reply:
Oh my bad 👁